package.json

```diff
@@ -187,7 +188,8 @@
   "scripts": {
     "build-assets": "webpack -p",
     "build-schema": "python manage.py get_graphql_schema > saleor/graphql/schema.graphql",
-    "generate-types": "apollo client:codegen --target=typescript types --globalTypesFile=saleor/static/dashboard-next/types/globalTypes.ts",
+    "build-types": "apollo client:codegen --target=typescript types --globalTypesFile=saleor/static/dashboard-next/types/globalTypes.ts",
+    "generate": "plop --plopfile .plop/plopfile.js",
```

Comment:
I think `npm run generate` sounds too enigmatic. I'd propose using a more explicit command like `generate-component` for that.

Reply:
At this moment it generates only components, but in the future, it can be used to generate other things.

Reply:
So how will we tell the generator what files to generate then? Still, I think `generate` doesn't give any guesses about what it actually does.

Reply:
Another idea: `npm run new-component`

Reply:
I agree with @maarcingebala with renaming it sth like `generate-component`.
OR u could leave generate, but change the prompt in such way, that ATM u could only choose creation of a new component.